### DISCUSSION TITLE
How to actually jump into the documentation popup?

### DISCUSSION THREAD

**Discussion by ghost** [hrsh7th/nvim-cmp#1969] [Q&A]
Completion suggestions  have this documentation popoup menu. 
![image](https://github.com/hrsh7th/nvim-cmp/assets/76608488/ac3813a9-a17a-4472-8495-0fedfd5bf223)

How can I make my cursor jump inside this popup? I know it can be scrolled but couldn't really find anything about entering it. I want to do this to be able to follow the links in the documentation. I found a older discussion on this topic but I didn't quite understand the solution described there.

Discussion in question: [discussion/1517](https://github.com/hrsh7th/nvim-cmp/discussions/1517)

**Comment by Shougo** [comment]
https://github.com/hrsh7th/nvim-cmp/discussions/1517#discussioncomment-5597884

**Reply by ghost** [reply]
Hey @Shougo, thanks for the link but I have already visited that discussion and mentioned it in my question. The proposed solution did not work for me and I would also like an explanation of the proposed solution as I did not understand it.

Cheers

**Comment by Shougo** [comment]
Hm...  I don't know the documentation window can be entered.

**Reply by ghost** [reply]
@Shougo 
Oh I see... In that case, its safe to say that a feature request is the way to go right? Just confirming before I pollute [issues](https://github.com/hrsh7th/nvim-cmp/issues)

**Reply by Shougo** [reply]
OK.  It seems feature.  The window cannot be entered.

Because 

```lua
---Open window
---@param style cmp.WindowStyle
window.open = function(self, style)
  if style then
    self:set_style(style)
  end

  if self.style.width < 1 or self.style.height < 1 then
    return
  end

  if self.win and vim.api.nvim_win_is_valid(self.win) then
    vim.api.nvim_win_set_config(self.win, self.style)
  else
    local s = misc.copy(self.style)
    s.noautocmd = true
    self.win = vim.api.nvim_open_win(self:get_buffer(), false, s)
    for k, v in pairs(self.opt) do
      opt.win_set_option(self.win, k, v)
    end
  end
  self:update()
end
```

`vim.api.nvim_open_win()` second argument is `enter` flag.  It is false.

```
nvim_open_win({buffer}, {enter}, {config})                   *nvim_open_win()*
```

**Reply by ghost** [reply]
Oh I see... makes sense... thanks a lot for your help... I'll make a feature request in [ideas](https://github.com/hrsh7th/nvim-cmp/discussions/categories/ideas) then

Link to feature request: https://github.com/hrsh7th/nvim-cmp/discussions/1970
